test/web/index.dart

```diff
@@ -14,6 +14,13 @@ void main() {
   });
 
   channel.stream.listen((s) {
-    channel.sink.add(s);
+    if (s.startsWith('send ')) {
+      var count = int.parse(s.split(' ').last);
+      for (var i = 1; i <= count; i++) {
```

Comment:
would seem more natural to make this zero based.

Reply:
Done.